war/src/main/js/api/securityConfig.js

```diff
@@ -1,13 +1,12 @@
 /**
  * Provides a wrapper to interact with the security configuration
  */
-
-var jenkins = require('../util/jenkins');
+import jenkins from '../util/jenkins';
```

Comment:
for consistency can you move the two requires up to import at the top?

Reply:
I have done it in this commit: https://github.com/jenkinsci/jenkins/pull/4425/commits/6cccb699697374b54470db95c2f6043168b88c8d

What do you think of the result?